### DISCUSSION TITLE
Can I use diffsitter together with delta?

### DISCUSSION THREAD

**Discussion by mawkler** [afnanenayet/diffsitter#1027] [Q&A]
Diffsitter looks really promising. Is it possible to use diffsitter together with [delta](https://github.com/dandavison/delta)? I would love to be able to use the diffsitter diffing algorithm while keeping delta's more ["graphical" highlighting](https://github.com/dandavison/delta?tab=readme-ov-file#a-syntax-highlighting-pager-for-git-diff-and-grep-output).

**Comment by afnanenayet** [comment]
Yeah I quite enjoy delta's UI too. I originally had plans to update diffsitter to have a more delta-like UI but it seems like some work so I've never really gotten around to it.

I've never actually tried to integrate with delta, I suppose if diffsitter outputs a standard diff format then delta should be able to parse it and give it a nice UI layer on top. I'm not sure how diffsitter will work with the standard diff format since it's line-by-line, and diffsitter will ignore all formatting when computing a diff.
